api/turing/service/mlp_service.go

```diff
@@ -3,17 +3,16 @@ package service
 import (
 	"context"
 	"fmt"
-	"log"
 	"net/http"
 	"time"
 
 	"github.com/antihax/optional"
 	merlin "github.com/gojek/merlin/client"
 	mlp "github.com/gojek/mlp/api/client"
+	"github.com/gojek/turing/api/turing/log"
```

Comment:
nit: `turing` related imports in a separate block?

Reply:
It's usually grouped by  standard library imports vs external dependencies imports, not by turing and non-turing deps